services/QuillLMS/app/services/user_pack_sequence_item_saver.rb

```diff
@@ -5,6 +5,7 @@ class UserPackSequenceItemSaver < ApplicationService
 
   class UserPackSequenceItemSaverError < StandardError; end
 
+  ENQUEUE_DELAY = Rails.configuration.x.user_pack_sequence_items.enqueue_delay
```

Comment:
New pattern that I'm using for [custom configuration](https://joyofrails.com/articles/mastering-custom-configuration-in-rails)

Rails docs [here](https://guides.rubyonrails.org/v7.1/configuring.html#custom-configuration)